Sources/SwiftDocC/Model/MarkdownOutput/Translation/MarkdownOutputSemanticVisitor.swift

```diff
@@ -0,0 +1,452 @@
+         
+        // Content
+        
+        markdownWalker.visit(Heading(level: 1, Text(symbol.title)))
+        markdownWalker.visit(symbol.abstract)
+        // Intentionally only including the primary declaration in the output, because we are only using the primary language. 
+        if let declarationFragments = symbol.declaration.first?.value.declarationFragments {
+            let declaration = declarationFragments
+                .map { $0.spelling }
+                .joined()
+            let code = CodeBlock(declaration)
+            markdownWalker.visit(code)
+        }
+        
+        if let parametersSection = symbol.parametersSection, parametersSection.parameters.isEmpty == false {
+            markdownWalker.visit(Heading(level: 2, Text(ParametersSection.title ?? "Parameters")))
+            for parameter in parametersSection.parameters {
+                markdownWalker.visit(Paragraph(InlineCode(parameter.name)))
+                markdownWalker.visit(container: MarkupContainer(parameter.contents))
+            }
+        }
+        
+        markdownWalker.visit(section: symbol.returnsSection)
+        
+        markdownWalker.visit(section: symbol.discussion, addingHeading: symbol.kind.identifier.swiftSymbolCouldHaveChildren ? "Overview" : "Discussion")
+        
+        markdownWalker.outgoingReferences = []
+        markdownWalker.withRenderingLinkList {
+            $0.visit(section: symbol.topics, addingHeading: "Topics")
+            $0.visit(section: symbol.seeAlso, addingHeading: "See Also")
+        }
+        
+        manifest?.relationships.formUnion(markdownWalker.outgoingReferences)
+        
+        for relationshipGroup in symbol.relationships.groups {
+            for destination in relationshipGroup.destinations {
+                switch context.resolve(destination, in: identifier) {
+                case .success(let resolved):
+                    add(target: resolved, type: .relatedSymbol, subtype: relationshipGroup.kind.manifestRelationship)
+                case .failure:
+                    if let fallback = symbol.relationships.targetFallbacks[destination] {
+                        add(fallbackTarget: fallback, type: .relatedSymbol, subtype: relationshipGroup.kind.manifestRelationship)
+                    }
+                }
+            }
+        }
+        // TODO: add support for missing sections rdar://166124742
+        return MarkdownOutputNode(metadata: metadata, markdown: markdownWalker.markdown)
+        
+    }
+}
+
+import SymbolKit
+
+private extension RelationshipsGroup.Kind {
+    var manifestRelationship: MarkdownOutputManifest.RelationshipSubType? {
+        // Structured like this to cause a compiler error if a new case is added
+        switch self {
+        case .conformingTypes: .conformingTypes
+        case .conformsTo: .conformsTo
+        case .inheritsFrom: .inheritsFrom
+        case .inheritedBy: .inheritedBy
+        }
+    }
```

Comment:
Question: If there's a 1 to 1 mapping between these types, why do we need to add a new type?